src/base/bittorrent/session.cpp

```diff
@@ -329,7 +329,15 @@ Session::Session(QObject *parent)
     , m_IPFilterFile(BITTORRENT_SESSION_KEY("IPFilter"))
     , m_announceToAllTrackers(BITTORRENT_SESSION_KEY("AnnounceToAllTrackers"), false)
     , m_announceToAllTiers(BITTORRENT_SESSION_KEY("AnnounceToAllTiers"), true)
+#if (LIBTORRENT_VERSION_NUM >= 20000)
+    , m_asyncIOThreads(BITTORRENT_SESSION_KEY("AsyncIOThreadsCount"), 3)
+    , m_hashingThreads(BITTORRENT_SESSION_KEY("HashingThreadsCount"), 1)
```

Comment:
Please follow the default values:
https://www.libtorrent.org/reference-Settings.html#aio_threads
https://www.libtorrent.org/reference-Settings.html#hashing_threads

Reply:
It's also probably a good time to address https://github.com/qbittorrent/qBittorrent/issues/11461; i.e., when using libtorrent < 2.0, the default async I/O threads should be `8` by default, which gives 2 hashing threads.